pkgs/top-level/darwin-packages.nix

```diff
@@ -83,10 +83,24 @@ impure-cmds // appleSourcePackages // chooseLibs // {
   };
 
   binutils-unwrapped = callPackage ../os-specific/darwin/binutils {
-    inherit (pkgs) binutils-unwrapped;
-    inherit (pkgs.llvmPackages) llvm clang-unwrapped;
+    inherit (self) cctools;
+    inherit (pkgs.llvmPackages) clang-unwrapped llvm llvm-manpages;
   };
 
+  # x86-64 Darwin gnat-bootstrap emits assembly
+  # with MOVQ as the mnemonic for quadword interunit moves
+  # such as `movq %rbp, %xmm0`.
+  # The clang integrated assembler recognises this as valid,
+  # but unfortunately the cctools^gas GNU assembler does not;
+  # it instead uses MOVD as the mnemonic.
+  # The assembly that a GCC build emits is determined at build time
+  # and cannot be changed afterwards.
+  #
+  # To build GNAT on x86-64 Darwin, therefore,
+  # we need both the clang _and_ the cctools^gas assemblers to be available:
```

Comment:
`^` is the notation Nix uses to refer to an output of a derivation. `nix build nixpkgs#darwin.cctools^gas` would build the `gas` output of cctools.

Reply:
> I think maybe this entire comment fits better on binutilsDualAs than on binutils?

I can move it down. Probably a copy-paste error moving it from the old darwin.binutils definition.

Reply:
Flake syntax? Surely that's experimental just like Flakes are : )

Reply:
I moved the comment and changed it to `cctools.gas`.

Reply:
The comment was moved but not changed but that shouldn't hold back the PR.

Reply:
It was there twice, and I’d only changed it in one place. I fixed it and force-pushed, which shouldn’t affect the hydra job because it’s just a comment that changed.